openmc/deplete/abc.py

```diff
@@ -861,12 +861,18 @@ def integrate(self, final_step=True):
             of the last timestep.
 
             .. versionadded:: 0.12.1
+        output : bool, optional
+            Indicate whether to display information about progress
```

Comment:
sometimes verbose is used instead of output for controlling additional terminal printing, I think output is fine but just worth mentioning for your consideration

Reply:
Indeed -- I used `output` mostly for consistency with similar methods in our Python API but perhaps we ought to switch to `verbose` at some point

Reply:
Yep let us leave this as ```output``` for now. I've raised an issue to see if people want to change it to ``verbose``` in the future #2038